src/Model/Location.php

```diff
@@ -60,17 +69,30 @@ public function formattedAddress() {
 	 * formattedAddressOneLine
 	 *
 	 * Returns the formatted Location address in one line, separated by comma
-	 *
-	 * @TODO: Do not return tags (,) if values are empty. This applies to  formattedAddress(), too
-	 *
 	 * @return string html
 	 */
-	public function formattedAddressOneLine() {
+	public function formattedAddressOneLine(): string {
+		$location_street = CB::get( \CommonsBooking\Wordpress\CustomPostType\Location::$postType, COMMONSBOOKING_METABOX_PREFIX . 'location_street', $this->post );
+
+		$location_postcode = CB::get( \CommonsBooking\Wordpress\CustomPostType\Location::$postType, COMMONSBOOKING_METABOX_PREFIX . 'location_postcode', $this->post );
+
+		$location_city = CB::get( \CommonsBooking\Wordpress\CustomPostType\Location::$postType, COMMONSBOOKING_METABOX_PREFIX . 'location_city', $this->post );
+
+		if (empty($location_street) && empty($location_postcode) && empty($location_city)){
+			return "";
+		}
+		elseif (empty($location_street) || empty($location_postcode)){
+			return sprintf('%s %s %s',
+				$location_city,
+				$location_postcode,
+				$location_city
```

Comment:
@hansmorb Komm gerade nicht klar, ob das so gewollt ist oder doch nurn Vertipper? Also, du prüfst, wenn Straße oder PLZ leer sind, dann zeige in folgender Reihenfolge:
Stadt PLZ Stadt
Wenn nur Straße leer ist würde dann ja: Köln 50968 Köln stehen, oder?
Wenn PLZ leer ist, dann nur noch Köln Köln

Reply:
Stimmt, danke, war nen Tippfehler.